priv/templates/nimble_template/.github/workflows/test.yml.eex

```diff
@@ -92,17 +93,21 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
 
-      - name: Set up Elixir
-        uses: erlef/setup-elixir@v1
-        with:
-          otp-version: ${{ env.OTP_VERSION }}
-          elixir-version: ${{ env.ELIXIR_VERSION }}
-      <%= if web_project? do %>
-      - name: Set up Node
-        uses: actions/setup-node@v2.1.5
+      - name: Setup asdf
+        uses: asdf-vm/actions/setup@v1
+
+      - name: Cache asdf
+        uses: actions/cache@v2
```

Comment:
I would give some comments similar to https://github.com/nimblehq/elixir-templates/pull/205#pullrequestreview-982524255
Please check it 🙏

Reply:
Adjusted in these commits @hoangmirs 
![image](https://user-images.githubusercontent.com/11751745/169940429-5e4c1cbb-bfda-4a32-ab97-59b8b90e43e9.png)